### PR TITLE
TBQ: * fix warnings

### PR DESCRIPTION
**Pull request by Tsurkan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
     </parent>
     <properties>
         <java.version>11</java.version>
+        <snakeyaml.version>1.33</snakeyaml.version>
```
Comment (Sekator778) [inline pom.xml:18]: what this?

File: src/main/java/ua/shpp/eqbot/command/AddService.java [hunk 2]
```diff
@@ -15,13 +14,14 @@
 import ua.shpp.eqbot.service.SendBotMessageService;
 
 import static ua.shpp.eqbot.model.PositionMenu.MENU_START;
+
```
Comment (Sekator778) [inline src/main/java/ua/shpp/eqbot/command/AddService.java:17]: why component 
what is the difference between component and service
i mean here service need

**Review by Sekator778 — COMMENTED** [review]
what is the main purpose of what merge?

File: pom.xml (hunk 1 above)
Comment (Tsurkan) [reply pom.xml:18]: There is a problem with dependencies in Spring. It is necessary to register the latest version of snakeyaml manually. And old people have many vulnerable places.

File: src/main/java/ua/shpp/eqbot/command/AddService.java (hunk 2 above)
Comment (Tsurkan) [reply src/main/java/ua/shpp/eqbot/command/AddService.java:17]: The code currently in the developer does not use...

**Pull request closed without merging**